### DISCUSSION TITLE
What's the difference between exposure and ev100?

### DISCUSSION THREAD

**Discussion by dontweaks** [google/filament#7984] [Q&A] [answered]
What's the difference between exposure and ev100?

```cpp
getPerViewUib...

  { "exposure",               0, Type::FLOAT,  Precision::HIGH, FeatureLevel::FEATURE_LEVEL_0 }, // high precision to work around #3602 (qualcom),
  { "ev100",                  0, Type::FLOAT,  Precision::DEFAULT, FeatureLevel::FEATURE_LEVEL_0 },
```

isn't ev100 "exposure value"?

**Comment by romainguy** [comment, marked as answer]
EV is exposure value which is different from photometric exposure commonly called exposure. The photetric exposure is what light intensities get multiplied by.

See https://github.com/google/filament/blob/main/filament/include/filament/Exposure.h and https://google.github.io/filament/Filament.md.html#imagingpipeline/physicallybasedcamera
